packages/website/src/components/hooks/useHashState.ts

```diff
@@ -95,7 +98,7 @@ const parseStateFromUrl = (hash: string): Partial<ConfigModel> | undefined => {
       code,
       fileType,
       eslintrc: eslintrc ?? '',
```

Comment:
If this modification is appropriate, I was wondering if it would be okay to specify initialState for eslintrc as well?

```suggestion
      eslintrc: eslintrc ?? initialState.eslintrc,
```

Reply:
Yeah good call!

Reply:
Thank you for checking :) I've added it.
4f3b3dc